packages/dd-trace/src/priority_sampler.js

```diff
@@ -24,9 +24,9 @@ const DEFAULT_KEY = 'service:,env:'
 const defaultSampler = new Sampler(AUTO_KEEP)
 
 class PrioritySampler {
-  constructor (env, { sampleRate, rateLimit = 100, rules = [] } = {}) {
+  constructor (env, { sampleRate, rateLimit = 100, samplingRules = [] } = {}) {
```

Comment:
This change is unnecessary since it's in the sampler.

Reply:
Tbh I don't even remember changing this. Will change back